neqo-transport/src/connection/params.rs

```diff
@@ -93,6 +94,7 @@ impl Default for ConnectionParameters {
             outgoing_datagram_queue: MAX_QUEUED_DATAGRAMS_DEFAULT,
             incoming_datagram_queue: MAX_QUEUED_DATAGRAMS_DEFAULT,
             fast_pto: FAST_PTO_SCALE,
+            fuzzing_mode: false,
```

Comment:
Throughout, I have a preference for this to be called "fuzzing", rather than the more extra verbose "fuzzing_mode".